src/java/htsjdk/variant/variantcontext/GenotypeLikelihoods.java

```diff
@@ -72,6 +72,13 @@
     */
     private final static GenotypeLikelihoodsAllelePair[] PLIndexToAlleleIndex = calculatePLcache(MAX_ALT_ALLELES_THAT_CAN_BE_GENOTYPED);
 
+
+    /*
+    * a cache of the PL index to the anyploid alleles it represents over all possible numbers of alternate alleles
+    */
+    protected final static List<List<Integer>> plIndexToAlleleIndices = new ArrayList<List<Integer>>();
```

Comment:
This should be a `Map<Integer, List<Integer>>`, not a `List<List<Integer>>`, no? That way it is clear that you put something in the right place. regardless of order of insertion. Also it is much clearer then that the map is ploidy->plIndexToAlleleIndeciesList(of that ploidy)